src/VirtualClient/VirtualClient.Main/profiles/PERF-MYSQL-SYSBENCH-OLTP.json

```diff
@@ -27,12 +27,124 @@
                 "DatabaseName": "$.Parameters.DatabaseName",
                 "DatabaseScenario": "$.Parameters.DatabaseScenario",
                 "Role": "Client",
-                "Duration": "00:30:00",
+                "Duration": "02:00:00",
                 "RecordCount": 10000,
                 "Workload": "oltp_read_write",
                 "PackageName": "sysbench",
                 "SkipInitialize": "$.Parameters.SkipInitialize"
             }
+        },
```

Comment:
Thank you for adding this. Does all these work with all-configurations? in-memory and mix etc?

Reply:
yes it does * now * !